db/db_local/local_db_client.go

```diff
@@ -154,8 +154,90 @@ func (c *LocalDbClient) ContructSearchPath(ctx context.Context, requiredSearchPa
 	return c.client.ContructSearchPath(ctx, requiredSearchPath, searchPathPrefix, currentSearchPath)
 }
 
-func (c *LocalDbClient) GetSchemaFromDB(ctx context.Context, schemas []string) (*schema.Metadata, error) {
-	return c.client.GetSchemaFromDB(ctx, schemas)
+func (c *LocalDbClient) GetSchemaFromDB(ctx context.Context) (*schema.Metadata, error) {
+	// build a ConnectionSchemaMap object to identify the schemas to load
+	// (pass nil for connection state - this forces NewConnectionSchemaMap to load it)
+	connectionSchemaMap, err := steampipeconfig.NewConnectionSchemaMap()
+	if err != nil {
+		return nil, err
+	}
+	// get the unique schema - we use this to limit the schemas we load from the database
+	schemas := connectionSchemaMap.UniqueSchemas()
+	query := c.buildSchemasQuery(schemas)
+
+	acquireSessionResult := c.AcquireSession(ctx)
```

Comment:
why do we need to acquire session here, but the base implementation just calls 

```
connection, err := c.dbClient.Conn(ctx)
```

Reply:
This is because `dbClient` is not a public property in the `DbClient` struct